spring-batch/src/main/java/org/baeldung/conditionalflow/step/PrependingStdoutWriter.java

```diff
@@ -0,0 +1,30 @@
+package org.baeldung.conditionalflow.step;
+
+import org.springframework.batch.item.ItemWriter;
+
+import java.io.OutputStream;
+import java.util.List;
+
+public class PrependingStdoutWriter<T> implements ItemWriter<T> {
+    private String prependText;
+    private OutputStream writeTo;
```

Comment:
Can we use a Logger?

Reply:
See other comments about System.out - IMO it's appropriate here